lib/api/controllers/documentController.js

```diff
@@ -72,11 +73,14 @@ function DocumentController (kuzzle) {
     assertBodyAttributeType(request, 'ids', 'array');
     assertHasIndexAndCollection(request);
 
+    if (request.input.body.ids.length > kuzzle.config.server.maxMultiActionsCount) {
```

Comment:
I have tendency to agree with @dbengsch, but I find @scottinet 's remark pertinent.

I mean, it's interesting to be able to limit the size of a response from Kuzzle to the client. If a guy makes a naive `m*` request that results in a 2Mb response because of the huge volome of documents in ES, well... He may never receive the response.
Yet, `maxMultiActionsCount` doesn't mean `maxResponseSize`. It means `maxMultiActionsCount`. Which is very good to avoid overheading Kuzzle in terms of memory and computation.

IMHO, you're both right, but we shoudn't over-responsabilize the `maxMultiActionsCount` parameter. I propose we introduce another `maxResponseSize` parameter, probably interesting from a networking point of view, the implementation of which is not to be discussed here immediately, especially since I have no clue about how to do it.